arm-sql/2015-05-01/examples/DatabaseBlobAuditingCreateMax.json

```diff
@@ -0,0 +1,57 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "blobauditingtest-4799",
+    "serverName": "blobauditingtest-6440",
+    "databaseName": "testdb",
+    "blobAuditingPolicyName": "Default",
+    "api-version": "2015-05-01-preview",
+    "databaseBlobAuditingPolicy": {
+      "properties": {
+          "state": "Enabled",
+          "storageAccountAccessKey": "sdlfkjabc+sdlfkjsdlkfsjdfLDKFTERLKFDFKLjsdfksjdflsdkfD2342309432849328476458/3RSD==",
+          "storageEndpoint": "https://mystorage.blob.core.windows.net",
+          "retentionDays": 6,
+          "storageAccountSubscriptionId": "00000000-1234-0000-5678-000000000000",
+          "isStorageSecondaryKeyInUse": false,
+          "auditActionsAndGroups":["DATABASE_LOGOUT_GROUP","DATABASE_ROLE_MEMBER_CHANGE_GROUP", "UPDATE on database::TestDatabaseName by public"]
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/blobauditingtest-4799/providers/Microsoft.Sql/servers/blobauditingtest-6440/databases/testdb",
+        "name": "Default",
+        "type": "Microsoft.Sql/servers/databases/auditingSettings",
+        "kind": "V12",
+          "properties": {
+            "state":"Enabled",
+            "storageEndpoint": "https://mystorage.blob.core.windows.net",
+            "storageAccountAccessKey": "sdlfkjabc+sdlfkjsdlkfsjdfLDKFTERLKFDFKLjsdfksjdflsdkfD2342309432849328476458/3RSD==",
+            "retentionDays": 0,
+            "storageAccountSubscriptionId": "00000000-1234-0000-5678-000000000000",
+            "isStorageSecondaryKeyInUse": false,
+            "auditActionsAndGroups": ["DATABASE_LOGOUT_GROUP", "DATABASE_ROLE_MEMBER_CHANGE_GROUP", "UPDATE on database::TestDatabaseName by public"]
+        }
+      }
+    },
+    "201": {
+      "body": {
+        "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/blobauditingtest-4799/providers/Microsoft.Sql/servers/blobauditingtest-6440/databases/testdb",
+        "name": "Default",
+        "type": "Microsoft.Sql/servers/databases/auditingSettings",
+        "kind": "V12",
+          "properties": {
+            "state":"Enabled",
+            "storageEndpoint": "https://mystorage.blob.core.windows.net",
```

Comment:
Also in case these APIs are not reviewed by ARM/API review board, please get these reviewed.